cpmonitor/templates/circle-checked-filled.html

```diff
@@ -0,0 +1,14 @@
+<svg style="color:green"
```

Comment:
I just added the icons as html for now, but probably there is a more universal way for using icons from tabler (and also change the color of the icon). Shall I make another issue for that or do we anyway not need so many more icons?